package.json

```diff
@@ -110,6 +110,7 @@
     "stylelint-prettier": "^1.1.2",
     "stylelint-scss": "^3.18.0",
     "turbo": "^1.4.3",
+    "vite": "^4.5.2",
```

Comment:
you should not have vite dependency at the root package.json of the monorepo

Reply:
It is used by the script that runs the tests with playwright : it starts a local server and stop it once the test are done